apertium-uzb.uzb.lexc

```diff
@@ -90,17 +126,18 @@ Multichar_Symbols
 %<title%>                         !
 
 !! Tense / finite forms
-%<pres%>      ! for "жатыр", "тұр", "отыр" and "жүр"
+%<pres%>      ! for "yotir", "dir",..
 %<aor%>       ! -{E}
-%<past%>      ! -{G}{A}н
+%<past%>      ! -{G}{A}n
```

Comment:
It should be just `-gan` in Uzbek, no?

Reply:
I changed it to  -{G}an . As far as I understood, for verbs ending with "k" and "q", it should be -kan and -qan respectively, no?

Reply:
Yes, perfect.